src/Export/Scripts/ScriptResources/JewelModWeights.lua

```diff
@@ -0,0 +1,177 @@
+local strTypes = {
+    -- prefixes
+    "AreaOfEffect",
+    "GlobalPhysicalDamageReductionRatingPercent",
+    "ArmourBreak",
+    "Attackdamage",
```

Comment:
Can/should this be `AttackDamage`?

Reply:
Yep, typo from my end. Thanks for picking up on it. I'll submit a fix after work if not resolved by then

Reply:
Fixed in #578